### PR TITLE
Support array of paths in input

### PR DESCRIPTION
**Pull request by nessex**
Works based upon this undocumented feature of minimist: https://github.com/substack/minimist/issues/108

### CODE REVIEW

File: entrypoint.js [hunk 1]
```diff
@@ -54,7 +68,7 @@ Toolkit.run(async tools => {
     };
 
     try {
-      const result = await tools.runInWorkspace('/entrypoint.sh', [base_ref, pr_ref, path]);
+      const result = await tools.runInWorkspace('/entrypoint.sh', [base_ref, pr_ref].concat(paths));
```
Comment (codesuki) [inline entrypoint.js]:
```suggestion
      const result = await tools.runInWorkspace('/entrypoint.sh', [base_ref, pr_ref, ...paths]);
```

File: entrypoint.js [hunk 2]
```diff
@@ -4,9 +4,23 @@ shim("check-master");
...
+      break;
+    case 'array':
+      paths = tools.arguments.path
+      break;
+    default:
+    case 'undefined':
+      paths = [ '.*' ]
+      break;
+  }
+
```
Comment (codesuki) [inline entrypoint.js:12]:
```suggestion
let path = tools.arguments.path || '.*';
```

File: entrypoint.js [hunk 3]
```diff
@@ -4,9 +4,23 @@ shim("check-master");
 const {Toolkit} = require('actions-toolkit');
 
 Toolkit.run(async tools => {
-  const path = tools.arguments.path || '.*';
+  let paths = [];
+
+  switch(typeof tools.arguments.path) {
```
Comment (codesuki) [inline entrypoint.js]:
```suggestion
  if (!Array.isArray(paths)) { 
      paths = [paths];
  }
```

**Review by codesuki — CHANGES_REQUESTED** [review]

File: entrypoint.js (hunk 1 above)
Comment (nessex) [reply entrypoint.js]: :+1:

File: entrypoint.js (hunk 3 above)
Comment (nessex) [reply entrypoint.js]: Forgot about that issue with arrays, changed it to use `Array.isArray`.

File: entrypoint.js (hunk 2 above)
Comment (nessex) [reply entrypoint.js:12]: We no longer use `path`, don't think this is required. Have changed this area of the code slightly, let me know what you think.

**Pull request merged**